sdcm/sct_events.py

```diff
@@ -478,6 +477,9 @@ def start_events_device(log_dir, timeout=5):
     EVENTS_PROCESSES['EVENTS_FILE_LOOGER'].start(log_dir)
     EVENTS_PROCESSES['EVENTS_GRAFANA_ANNOTATOR'].start()
 
+    # default filters
+    DbEventsFilter(type='BACKTRACE', line='supressed')
```

Comment:
I think it is better to more specific: `line="Rate-limit: supressed"`

Reply:
BTW, they had a type in suppressed, I reported that in one of the issues, so let's have both in the filter, in case they'll fix it.

Reply:
fixed